### PR TITLE
Request proof of work for purchasing brokernode addresses

### PR DESCRIPTION
**Pull request by EdmundMai**
Step #2 from here: https://docs.google.com/document/d/1FCS_55lhRd539xl2FEQuIZvzxQDslGhWPsPBEUDLtEk/edit

- Deletes legacy folder

### CODE REVIEW

File: actions/transaction_brokernodes.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+				TrunkTx:  dataMap.TrunkTx,
+			},
+		}
+
+		return c.Render(200, r.JSON(res))
+	}
+}
+
+// TODO: put this in a helper
+func join(A []string, delim string) string {
```
Comment (EdmundMai) [inline actions/transaction_brokernodes.go]: Where should I put this? This is just a generic `.join` on arrays

File: actions/transaction_brokernodes.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+				Status:    "PAYMENT_PENDING",
+				DataMapID: dataMap.ID,
+				Purchase:  brokernode.Address,
+			}
+			tx.ValidateAndSave(&t)
+			return nil
+		})
+
+		res := transactionCreateRes{
+			ID: t.ID,
```
[the remark this answers is not in the document]
Comment (EdmundMai) [reply actions/transaction_brokernodes.go]: Yeh, it's automatically generated

File: actions/upload_sessions.go [hunk 3]
```diff
@@ -3,9 +3,10 @@ package actions
 import (
 	"bytes"
 	"encoding/json"
+	"fmt"
 	"net/http"
 
-	raven "github.com/getsentry/raven-go"
```
Comment (AaronVasquez) [inline actions/upload_sessions.go]: Undo this (your force push overwrote it)

File: actions/upload_sessions.go [hunk 4]
```diff
@@ -84,6 +85,8 @@ func (usr *UploadSessionResource) Create(c buffalo.Context) error {
 	}
 
 	if len(vErr.Errors) > 0 {
+		fmt.Println("xxxxxxxxxxxxxxxxxxxxxxxxX")
```
Comment (AaronVasquez) [inline actions/upload_sessions.go]: Remove

**Pull request merged**